src/pie-chart/index.tsx

```diff
@@ -46,7 +46,18 @@ const PieChart = function PieChart<T extends PieChartProps.Datum = PieChartProps
   detailPopoverSize = 'medium',
   ...props
 }: PieChartProps<T>) {
-  const { __internalRootRef = null } = useBaseComponent('PieChart');
+  const { __internalRootRef = null } = useBaseComponent('PieChart', {
+    props: {
+      fitHeight,
```

Comment:
Note: this component is missing in the doc, but I see that these are the props of it that meet the given criteria.

Reply:
Yes, and this is not the only one missing. I fill the gaps as I go through our entire repo